File: test/fixtures/multiresolve/android/src/main/java/com/example/multiresolve/MultiPlugin.java
package com.example.multiresolve;

import com.getcapacitor.JSObject;
import com.getcapacitor.Plugin;
import com.getcapacitor.PluginCall;
import com.getcapacitor.PluginMethod;
import com.getcapacitor.annotation.CapacitorPlugin;

@CapacitorPlugin(name = "Multi")
public class MultiPlugin extends Plugin {
    @PluginMethod
    public void getA(PluginCall call) { call.resolve(new JSObject().put("a", "x")); }
    @PluginMethod
    public void getB(PluginCall call) { call.resolve(new JSObject().put("b", 2)); }
    @PluginMethod
    public void getC(PluginCall call) { call.resolve(new JSObject().put("c", true)); }
}

File: test/fixtures/multiresolve/ios/Sources/MultiPlugin.swift
import Foundation
import Capacitor

@objc(MultiPlugin)
public class MultiPlugin: CAPPlugin, CAPBridgedPlugin {
    public let identifier = "MultiPlugin"
    public let jsName = "Multi"
    public let pluginMethods: [CAPPluginMethod] = []

    @objc func getA(_ call: CAPPluginCall) { call.resolve(["a": "x"]) }
    @objc func getB(_ call: CAPPluginCall) { call.resolve(["b": 2]) }
    @objc func getC(_ call: CAPPluginCall) { call.resolve(["c": true]) }
}
